frontend/src/main.ts

```diff
@@ -2,9 +2,16 @@ import Vue from "vue";
 import App from "./App.vue";
 import router from "./router";
 import store from "./store";
+import * as Sentry from "@sentry/browser";
+import * as Integrations from "@sentry/integrations";
 
 Vue.config.productionTip = false;
 
+const DNS = process.env.VUE_APP_DNS || "";
```

Comment:
DNS -> DSN :stuck_out_tongue:

Reply:
Hmm, burde kanskje være mer spesifikk om hva slags DNS det er snakk om 😛

Reply:
Haha, mener reffefølgen på bokstavene :sweat_smile: 
DSN, for _Data Source Name_

Reply:
Yikes, skal fikse det 👍

Reply:
Haha, har tatt meg selv i å skrive det feil før selv :P